### PR TITLE
Add newsletter sponsorship prospectus

### PR DESCRIPTION
**Pull request by ericholscher**

### CODE REVIEW

File: docs/newsletter.rst [hunk 1]
```diff
@@ -3,16 +3,22 @@ Newsletter & Mailing lists
...
 
-You can also subscribe to our Conference mailing lists to stay up-to-date on upcoming events.
-
 If you have any questions or want to contribute to the newsletter, go ahead and email us at `newsletter@writethedocs.org
 <mailto:newsletter@writethedocs.org>`_.
 
+Sponsorship
+-----------
+
+We have added the ability to sponsor our newsletter.
```
Comment (plaindocs) [inline docs/newsletter.rst]:
```suggestion
As of June 2019 you can :doc:`</newsletter/sponsorship/> sponsor the newsletter` on a monthly basis.
```

File: docs/newsletter.rst [hunk 2]
```diff
@@ -3,16 +3,22 @@ Newsletter & Mailing lists
...
+You can read more at our :doc:`/newsletter/sponsorship/` page.
```
Comment (plaindocs) [inline docs/newsletter.rst]:
```suggestion

```

File: docs/newsletter/sponsorship.rst [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+Our audience is made up of technical writers and decision makers in the software documentation industry.
+The current newsletter stats are:
+
+* Over 5,000 subscribers
+* 37% open rate
+* 6% click rate
+* 70% of subscribers are based in the United States
+
+We don't have specific demographic data for our newsletter,
+but our conferences have:
```
Comment (plaindocs) [inline docs/newsletter/sponsorship.rst]:
```suggestion
but our conferences are made up of:
```

**Review by plaindocs — APPROVED** [review]
Looks good.

File: docs/newsletter.rst [hunk 4]
```diff
@@ -3,16 +3,21 @@ Newsletter & Mailing lists
...
 interesting information that has been shared in our Slack network into the
-newsletter. The primary links you need are:
+newsletter. 
+
+The primary links you need are:
 
 * Our full `newsletter archives </blog/archive/tag/newsletter/>`__ 
 * Our `index <#index-of-newsletter-topics>`__ which sorts our writing by topic 
 
-You can also subscribe to our Conference mailing lists to stay up-to-date on upcoming events.
```
Comment (bethaitman) [inline docs/newsletter.rst:11]: Does this mean we're also removing the conference mailing list sign-up from this page?

File: docs/newsletter/sponsorship.rst [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+but our conferences are made up of:
+
+- Technical Writers (60%)
+- Developers (10%)
+- Support Staff (10%)
+- Managers (10%)
+- Community Contributors, Enthusiasts & Other Folks (10%)
+
+
+Format
```
Comment (bethaitman) [inline docs/newsletter/sponsorship.rst]:
```suggestion
Sponsored entry format
```

Maybe? To be a bit clearer

File: docs/newsletter/sponsorship.rst [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+------
+
+The current format is:
```
Comment (bethaitman) [inline docs/newsletter/sponsorship.rst:37]:
```suggestion
The current format of a sponsored entry in the newsletter is:
```

File: docs/newsletter/sponsorship.rst [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+
+* 300x300 pixel logo image
+* Up to 300 words of copy
```
Comment (bethaitman) [inline docs/newsletter/sponsorship.rst]: 300 words is a _lot_ - most of the articles are under 250, so I'd prefer to make this a little shorter.

File: docs/newsletter/sponsorship.rst [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+* Up to 2 links in that copy
+
+We're still working out our exact format details,
```
Comment (bethaitman) [inline docs/newsletter/sponsorship.rst]:
```suggestion
We're still experimenting with exact format details,
```

**Review by bethaitman — COMMENTED** [review]
A few small comments, but in general this looks reasonable!

File: docs/newsletter.rst (hunk 4 above)
Comment (ericholscher) [reply docs/newsletter.rst:11]: I forgot we combine these. We should break the conference mailing lists out into their own section. I can keep this for now, and do that later.

File: docs/newsletter/sponsorship.rst (hunk 7 above)
Comment (ericholscher) [reply docs/newsletter/sponsorship.rst]: Sounds good. I was thinking characters not words (from advertising heh) -- let's do 100 words?

Comment (ericholscher) [reply docs/newsletter/sponsorship.rst]: I went with 300 *characters*, since we care more about the space it takes than the number of words.

Comment (bethaitman) [reply docs/newsletter/sponsorship.rst]: Sure, let's try it. If people want longer they can just tell us and we can talk about it when we have a specific example!

File: docs/newsletter.rst (hunk 4 above)
Comment (bethaitman) [reply docs/newsletter.rst:11]: Sounds good, thanks!

**Review by bethaitman — APPROVED** [review]

**Comment by ericholscher** [comment]
Shipping it!

**Pull request merged**